app/spec/instance-event/commons.spec.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+describe("Commons for instance events", function() {
+  const fs = require("fs");
+  const path = require("path");
+
+  const config = require(path.resolve("src/utils/config"));
+
+  const cmns = require(path.resolve(
+    "src/" + config.getJobNameForInstanceEvent() + "/commons"
+  ));
+
+  it("should substitute aliases present in an instance definition.", function() {
+    var instanceDef = JSON.parse(
+      fs.readFileSync(
+        path.resolve(
+          "spec/instance-event/resources/test_instance_definition_2.json"
+        ),
+        "utf8"
+      )
+    );
+
+    instanceDef.uuid = "e1eddb12-96c3-4afc-a116-d2008f86e0a3";
+
+    var substitutedInstanceDef = cmns.substituteAliases(instanceDef);
```

Comment:
If you decide to adopt the test structure setup - replay - verif - after, then replay is the most important part of the test to be singled out. I would definitely precede this specific instruction with `//replay`.